frontend/src/Pages/FC/Bans.js

```diff
@@ -140,10 +144,17 @@ function AddBan() {
           Duration in minutes (0 for permanent)
           <br />
         </label>
-        <Input type="number" value={duration} onChange={(evt) => setDuration(evt.target.value)} />
+          <Input type="number" value={duration} onChange={(evt) => setDuration(evt.target.value)} />
       </p>
-      <Button variant="danger" onClick={(evt) => doBan()}>
-        Ban
+        <p>
+            <label>
+                Ban Reason
+                <br />
+            </label>
+            <Input value={banReason} onChange={(evt) => setBanReason(evt.target.value)} />
```

Comment:
Since the backend only allows 255 characters, please restrict it to that many here already